### PR TITLE
[DEVREL-1179] Fix: Borken Links | App-Dev-Features

### PR DESCRIPTION
**Pull request by teja-kummarikuntla**
Fixing Issue #330

### CODE REVIEW

File: Freshworks-Samples/App-Development-Features/README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ Platform feature | Sample code | Notes
...
+[Installation Parameters](https://medium.com/freshworks-developer-blog/securing-sensitive-installation-parameters-3879908ade17) | [Config iparams JSON](https://github.com/freshworks/marketplace-sample-apps/tree/master/Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparams-json-freshdesk/config), [Config iparams HTML](https://github.com/freshworks/marketplace-sample-apps/tree/master/Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparams-html-freshdesk/config) | Common pattern across all products
 [Custom Installation Page](https://medium.com/freshworks-developer-blog/updates-to-the-custom-installation-page-b787b66c8a39) | [Custom Installation page demo App](https://github.com/freshdesk/marketplace-sample-apps/tree/master/Freshdesk/v2/custom_installation_page_app) | NaN
```
Comment (teja-kummarikuntla) [inline Freshworks-Samples/App-Development-Features/README.md:16]: [Custom Installation Page](https://medium.com/freshworks-developer-blog/updates-to-the-custom-installation-page-b787b66c8a39), A few hyperlinks in this Blog content are broken

Comment (teja-kummarikuntla) [reply Freshworks-Samples/App-Development-Features/README.md:16]: > [Custom Installation page demo App](https://github.com/freshdesk/marketplace-sample-apps/tree/master/Freshdesk/v2/custom_installation_page_app) [BROKEN]

I'm unsure which app URL in the repo helps replacing, `config-iparams-json-freshdesk` and `config-iparams-html-freshdesk` are already used above

**Review by teja-kummarikuntla — COMMENTED** [review]
@Saif-Shines , I've pointed a few questions as reviews below, we can have another commit to fix them prior to merging.

File: Freshworks-Samples/App-Development-Features/README.md (hunk 1 above)
Comment (Saif-Shines) [reply Freshworks-Samples/App-Development-Features/README.md:16]: Can we say, Build configuration pages using HTML or JSON? 

Where HTML and JSON will by hyperlinked to respective apps in the repository?

**Review by Saif-Shines — CHANGES_REQUESTED** [review]
Thanks for fixing this.

**Review by Saif-Shines — APPROVED** [review]

**Comment by kaustavdm** [comment]
Just saying, I love that the title says, "borken links". It's very doge. :-D

**Pull request merged**